src/Form/IssueType.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\DataProvider;
+use App\Entity\Issue;
+use App\Entity\Project;
+use App\Entity\Version;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class IssueType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name')
```

Comment:
I don't think any of these fields should be editable, so this file seems superfluous.

Reply:
You're right. It has been autogenerated and will be removed manually.